### PR TITLE
Bugfix/register token without org

### PR DESCRIPTION
**Pull request by vaimdev**
The fix expected to fix issue https://github.com/graphistry/pygraphistry/issues/488
and also other issue when register() with JWT token.

### CODE REVIEW

File: graphistry/pygraphistry.py [hunk 1]
```diff
@@ -382,6 +377,11 @@ def refresh(token=None, fail_silent=False):
                 PyGraphistry._is_authenticated = True
             return PyGraphistry.api_token()
         except Exception as e:
+            if PyGraphistry.store_token_creds_in_memory():
```
Comment (lmeyerov) [inline graphistry/pygraphistry.py:380]: This will not actually refresh?

**Review by lmeyerov — CHANGES_REQUESTED** [review]
See comment, fix breaks refresh

File: graphistry/pygraphistry.py (hunk 1 above)
Comment (lmeyerov) [reply graphistry/pygraphistry.py:380]: Store token creds makes a login() method that can be called, while api-token() returns the stale eve if stale

Comment (lmeyerov) [inline graphistry/pygraphistry.py:380]: on `if` of 375 if statement, should we add an `else: return token` ?

File: graphistry/pygraphistry.py [hunk 2]
```diff
@@ -382,6 +377,11 @@ def refresh(token=None, fail_silent=False):
...
+                logger.debug("JWT refresh via creds")
+                logger.debug("2. @PyGraphistry refresh :relogin")
+                return PyGraphistry.relogin()
+
```
Comment (lmeyerov) [inline graphistry/pygraphistry.py:384]: shouldn't it be `if not fail_silent: ... raise e` ?

**Comment by lmeyerov** [comment]
- [ ] @lmeyerov  TODO: For review acceptance, should test against removing https://github.com/graphistry/graphistrygpt/blob/1e745e52c4ebc4eff6b58b22547805d1efff8d53/graphistrygpt/utils.py#L43 in multi-user scenarios

File: graphistry/pygraphistry.py (hunk 2 above)
Comment (vaimdev) [reply graphistry/pygraphistry.py:384]: fixed

File: graphistry/pygraphistry.py (hunk 1 above)
Comment (lmeyerov) [reply graphistry/pygraphistry.py:380]: @vaimdev how about the comment ^^^ on missing `else: return token` ? a bug to fix or correct as is?

The implicit `else` case currently ignores `token`, returning instead stale, right?

**Pull request merged**